src/ReaderThemeSupportFeatures.php

```diff
@@ -203,13 +256,35 @@ public function update_cached_theme_support() {
 	 */
 	public function get_theme_support_features( $reduced = false ) {
```

Comment:
There is missing test coverage in this method. Is that feasible to add?

Reply:
Tests are there but they were wrongfully skipped due to https://github.com/ampproject/amp-wp/pull/7481/commits/f1d16af946f18b3f2e9d44fb42e7724be265ffa3.

I have fixed it in https://github.com/ampproject/amp-wp/pull/7481/commits/722476c6b1c61b26b08eda42d954a95e115398c5